### PR TITLE
Test/8279 fix backup seed phrase

### PR DESCRIPTION
**Pull request by StateOf-Vlado**
### What does the PR do

Fixes https://github.com/status-im/status-desktop/issues/8279

### StatusQ checklist

- [ ] add documentation if necessary (new component, new feature)
- [ ] update sandbox app
  - in case of new component, add new component page
  - in case of new features, add variation to existing component page
  - nice to have: add it to the demo application as well
- [ ] test changes in both light and dark theme?

### Screenshot of functionality (including design for comparison)

- [ ] I've checked the design and this PR matches it

### Cool Spaceship Picture

<!-- optional but cool ->

### CODE REVIEW

**Comment by anastasiyaig** [comment]
@StateOf-Vlado hey, thank you for the contribution! Some comments from me:

- commits should be squashed to 1
- there should be a reference to a ticket / task if exists
- good to have PR description filled in with actual description what has been done :)

**Comment by anastasiyaig** [comment]
also, the test is failing to me with:

```
Detail LookupError: Object '{container={container={name='mainWindow' type='StatusWindow' visible='true'} type='Overlay' unnamed='1' visible='true'} objectName?='ConfirmSeedPhrasePanel_StatusSeedPhraseInput_1' type='StatusSeedPhraseInput' visible='true'}' not found. Could not match properties:
    objectName,type for object name: '{container={container=:statusDesktop_mainWindow type='Overlay' unnamed='1' visible='true'} objectName?='ConfirmSeedPhrasePanel_StatusSeedPhraseInput_1' type='StatusSeedPhraseInput' visible='true'}'  /Users/anastasiya/status-desktop/test/ui-test/src/drivers/SquishDriver.py:163
```

**Review by anastasiyaig — CHANGES_REQUESTED** [review]
test is failing to me with:

```
Detail LookupError: Object '{container={container={name='mainWindow' type='StatusWindow' visible='true'} type='Overlay' unnamed='1' visible='true'} objectName?='ConfirmSeedPhrasePanel_StatusSeedPhraseInput_1' type='StatusSeedPhraseInput' visible='true'}' not found. Could not match properties:
    objectName,type for object name: '{container={container=:statusDesktop_mainWindow type='Overlay' unnamed='1' visible='true'} objectName?='ConfirmSeedPhrasePanel_StatusSeedPhraseInput_1' type='StatusSeedPhraseInput' visible='true'}'  /Users/anastasiya/status-desktop/test/ui-test/src/drivers/SquishDriver.py:163
```

**Comment by StateOf-Vlado** [comment]
@anastasiyaig 
**good to have PR description filled in with actual description what has been done**, do you mean technical description, like mouse hover on the button added to fix click on the hidden element?
Or do I need to copy/paste the description from the task?

**Comment by StateOf-Vlado** [comment]
Test results: https://ci.status.im/job/status-desktop/job/branches/job/linux/job/x86_64/job/tests-e2e/job/PR-10204/9/squishResults_230411_164354.592/webreport_230411_164354.592_linux-02/index.html

File: test/ui-test/src/screens/StatusMainScreen.py [hunk 1]
```diff
@@ -193,3 +194,10 @@ def navigate_to_edit_profile(self):
     def close_popup(self):
         # Click in the corner of the overlay to close the popup
         click_obj_by_name_at_coordinates(MainScreenComponents.POPUP_OVERLAY.value, 1, 1)
+    
+    def is_secure_your_seed_phrase_banner_visible(self, value: bool):
+        verify(
+            squish.waitFor(lambda: is_found(MainScreenComponents.SECURE_YOUR_SEED_PHRASE_BANNER.value) is value, 15 * 1000),
```
Comment (noeliaSD) [inline test/ui-test/src/screens/StatusMainScreen.py:200]: Here we don't directly use `squish API`, we go through the `SquishDriver` instead to separate the automation tool from the screen's classes. If we already don't have a wrapper for the `waitFor` in the `SquishDriver`, I suggest you to add it.

**Review by noeliaSD — CHANGES_REQUESTED** [review]
I've tested locally and it consistently fails for me. I'm testing on `mac os` just in case it is relevant.

See the video here:
https://user-images.githubusercontent.com/97019400/231488595-f90fb563-c836-4bba-8749-56b013bd2f50.mov

File: test/ui-test/src/screens/StatusMainScreen.py (hunk 1 above)
Comment (StateOf-Vlado) [reply test/ui-test/src/screens/StatusMainScreen.py:200]: should we move 'verify' there also, because it is also squish method?

Comment (StateOf-Vlado) [reply test/ui-test/src/screens/StatusMainScreen.py]: fixed

**Review by noeliaSD — APPROVED** [review]
I cannot ensure it works, since it still fails for me locally consistently. However, since in CI it passes, code LGTM so, I approve.

Please @jrainville @alaibe could some of you also test it locally?

File: test/ui-test/src/screens/StatusMainScreen.py [hunk 2]
```diff
@@ -193,3 +194,9 @@ def navigate_to_edit_profile(self):
     def close_popup(self):
         # Click in the corner of the overlay to close the popup
         click_obj_by_name_at_coordinates(MainScreenComponents.POPUP_OVERLAY.value, 1, 1)
+    
+    def is_secure_your_seed_phrase_banner_visible(self, value: bool):
+        verify(is_found(MainScreenComponents.SECURE_YOUR_SEED_PHRASE_BANNER.value) is value,
```
Comment (noeliaSD) [inline test/ui-test/src/screens/StatusMainScreen.py:199]: Verify is ok since it is using the `SquishDriverVerification` api.

**Pull request merged**